Assets/Project/Scripts/Common/Entities/GameDatas/GimmickData.cs

```diff
@@ -42,22 +42,22 @@ public class GimmickData
         public List<int> randomDirection;
 
         /// <summary>
-        /// ＜竜巻｜隕石＞ランダム攻撃行重みリスト
+        /// ＜竜巻｜隕石｜霧＞ランダム攻撃行重みリスト
```

Comment:
`攻撃` って文言がもはや変なので，
```suggestion
        /// ＜竜巻｜隕石｜霧＞ランダム行重みリスト
```
にしますか？

Reply:
これはあり